src/utils/formatting/stringFormatting.test.js

```diff
@@ -67,152 +68,232 @@ describe('formatValueFromStateAndProps', () => {
       valueFormat,
       optionalValueFormat
     });
+    selectors.stockOverviewDataSelector =
+      jest.fn().mockReturnValue(state.stocks.MSFT.stockOverviewData);
+
     expect(formatValueFromStateAndProps(state, ownProps))
       .to
       .equal(`${expectedValueFormatted} (${expectedOptionalValueFormatted})`);
+
+    jest.resetAllMocks();
```

Comment:
Do `jest.fn().mockReturnValue(state.stocks.MSFT.stockOverviewData);` in `beforeAll` and then you should be able to remove `selectors.stockOverviewDataSelector = jest.fn().mockReturnValue(state.stocks.MSFT.stockOverviewData);` and ` jest.resetAllMocks();` from each test?